### PR TITLE
Issue204 - ssgen download

### LINKED ISSUE #204: Download SampleSheet to direct location
Option to download samplesheet to local PC or direct to specified location (needed for movement of samba share to webserver)

### PR DESCRIPTION
**Pull request by SophieS9**
Option to save samplesheet locally or download directly to webserver. Not sophisticated but works! Needs folder structure as per mysite/settings.py. 

Unit tests run with no issues.

Test data on green harddrive including most recent fixtures for setting up ssgen. 

Closes #204.

### CODE REVIEW

File: mysite/settings.py
Comment (Istaisa) [inline mysite/settings.py:1]: We'll just need to remember to change this gen01 to local, or flip the if statement and do if deploy_location == local

**Review by Istaisa — COMMENTED** [review]
Tested locally for a variety of run types and combinations, functionality works to download to webserver. 
Only suggestion/comment is that if you download a samplesheet again and there's already one of that name it overwrites the previous, this might be more of a problem for combination worksheets where the folder name is worksheet1 rather than worksheet1_worksheet2.

**Comment by SophieS9** [comment]
Updated so that folder name is now a combination of all worksheets - pretty simple change!

**Review by Istaisa — APPROVED** [review]
Folder names for joint worksheets now includes both options. Code works as expected, unit tests run, happy to merge.

**Pull request merged**